packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Inputs/index.tsx

```diff
@@ -214,24 +217,58 @@ const Inputs = () => {
 
                                 if (amountLimits) {
                                     const amount = Number(value);
-                                    if (amountLimits.minCrypto && amount < amountLimits.minCrypto) {
+                                    const btcAccountAndSatsUsed =
+                                        account.symbol === 'btc' && areSatsUsed;
+
```

Comment:
I think sats  can be used also in testnet and regtest and in the future additional coins could be enabled too. I would not hardcode the functionality to a specific coin. Isn't `areSatsUsed` enough as the condition variable?

Reply:
Definitely a no-go :( The source of truth for the amount units is the [networks config - features](https://github.com/trezor/trezor-suite/blob/2a985c667fe263912e229367b76a8e7596c0cae8/suite-common/wallet-config/src/networksConfig.ts#L15)

Also, the `useBitcoinAmountUnit` hook accept the symbol and returns `areUnitsSupportedByNetwork` if needed

Reply:
This piece of code was written before I got to this task. @dahaca, as Martin asks, is it sufficient to rely on `areSatsUsed`?

Reply:
Yes `areSatsUsed` is `areSatsDisplayed && areUnitsSupportedByNetwork && areUnitsSupportedByDevice`, so it covers all possible reasons for them not to be used. BTW it's in the context of using satoshis for wallet operations (send, trade, etc)